components/content_settings/renderer/brave_content_settings_agent_impl.cc

```diff
@@ -270,18 +271,22 @@ bool BraveContentSettingsAgentImpl::IsCosmeticFilteringEnabled(
 
 bool BraveContentSettingsAgentImpl::IsFirstPartyCosmeticFilteringEnabled(
     const GURL& url) {
+  if (!IsCosmeticFilteringEnabled(url))
```

Comment:
absense of this was a bug?

Reply:
They were added as part of a patch you had provided me. I have tested, and I can confirm that check in particular is not necessary and no tests are failing without it. I'm removing it in e5499c4da52e075b3e19d0077bed8377f9aab08d